app/src/androidTest/java/com/android/voyageur/ui/trip/schedule/ByDayScreenTest.kt

```diff
@@ -30,17 +29,18 @@ class ByDayScreenTest {
     composeTestRule.setContent { ByDayScreen(sampleTrip, navigationActions) }
     composeTestRule.onNodeWithTag("byDayScreen").assertIsDisplayed()
     composeTestRule.onNodeWithTag("bottomNavigationMenu").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("emptyByDayPrompt").assertIsDisplayed()
-  }
-
-  @Test
-  fun displaysCorrectTripName() {
-    composeTestRule.setContent { ByDayScreen(sampleTrip, navigationActions) }
-    composeTestRule
-        .onNodeWithTag("emptyByDayPrompt")
-        .assertTextContains(
-            "You're viewing the ByDay screen for Sample Trip, but it's not implemented yet.")
+    // composeTestRule.onNodeWithTag("emptyByDayPrompt").assertIsDisplayed()
```

Comment:
nit: in the future please try do delete these lines instead of commenting them, since it makes code more readable

Reply:
Thanks for the suggestion! I'll keep this in mind and delete in the next PR.